proof/refine/ARM/Ipc_R.thy

```diff
@@ -4085,27 +4275,7 @@ lemma valid_tcb_state'_update[simp]:
 crunches addToBitmap, setQueue
   for valid_tcb_state'[wp]: "valid_tcb_state' ts"
 
-lemma setSchedContext_obj_at':
-  "\<lbrace>\<lambda>s. sc_at' p s \<longrightarrow> P (obj_at' (if p = p' then \<lambda>_. P' v else P') p' s)\<rbrace>
-   setSchedContext p v
-   \<lbrace>\<lambda>rv s. P (obj_at' P' p' s)\<rbrace>"
-  unfolding obj_at'_real_def
-  apply (wp set_sc'.ko_wp_at)
-  apply (auto simp: obj_at'_def ko_wp_at'_def projectKO_eq projectKO_reply)
-  done
-
-lemma setReply_obj_at':
-  "\<lbrace>\<lambda>s. reply_at' rptr s \<longrightarrow> P (obj_at' (if rptr = p' then \<lambda>_. Q r else Q) p' s)\<rbrace>
-    setReply rptr r
-   \<lbrace>\<lambda>rv s. P (obj_at' Q p' s)\<rbrace>"
-  apply (simp only: obj_at'_real_def)
-  apply (wp set_reply'.ko_wp_at)
-  apply (auto simp: obj_at'_def ko_wp_at'_def projectKO_eq projectKO_reply)
-  done
-
-lemma setObject_reply_pde_mappings'[wp]:
-  "setObject ptr (val :: reply) \<lbrace>valid_pde_mappings'\<rbrace>"
-  by (wp valid_pde_mappings_lift')
+crunches bindScReply for irq_node'[wp]: "\<lambda>s. P (irq_node' s)"
```

Comment:
Don't we normally put the `for blah` bit on a new line? I'd consult the style guide, IF THERE WERE ONE